modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/FSMCallerImpl.java

```diff
@@ -497,6 +497,17 @@ private void doCommitted(final long committedIndex) {
                 final LogEntry logEntry = iterImpl.entry();
                 if (logEntry.getType() != EnumOutter.EntryType.ENTRY_TYPE_DATA) {
                     if (logEntry.getType() == EnumOutter.EntryType.ENTRY_TYPE_CONFIGURATION) {
+                        ConfigurationEntry configurationEntry = new ConfigurationEntry(
+                                logEntry.getId().copy(),
+                                new Configuration(logEntry.getPeers(), logEntry.getLearners()),
+                                null
```

Comment:
Looks like `ConfigurationEntry` does not expect to be provided with `null` here, you should use an empty configuration instead